eralchemy/sqla.py

```diff
@@ -23,6 +22,8 @@ def relation_to_intermediary(fk):
 
 def format_type(typ):
     """ Transforms the type into a nice string representation. """
+    from sqlalchemy.exc import CompileError
```

Comment:
I don't think it will add a significant amount of overhead since Python caches modules once they are imported the first time, but it might hurt a tiny bit (~400 ns). See [here](https://stackoverflow.com/questions/29504313/python-what-is-the-cost-of-re-importing-modules) and [here](https://wiki.python.org/moin/PythonSpeed/PerformanceTips#Import_Statement_Overhead).